config/config_win.yaml

```diff
@@ -0,0 +1,48 @@
+common:
+  daemonize: true
+  pid_dir: /var/run/
+  log_media: file
+  log_level: info
+  log_dir:  C:\Program Files\Crowdsec\log\
+  working_dir: .
+config_paths:
+  config_dir:  C:\Program Files\Crowdsec\config\
```

Comment:
You may want to consider defaulting configs, dynamic data, logs etc... to a subfolder of C:\ProgramData instead of writing to Program Files. AppData would normally be recommended, but since it looks like you're using a windows service per system model rather than a per user model, ProgramData would likely make more sense.

Reply:
Thanks for the suggestion !